tests/RegionT.cpp

```diff
@@ -541,28 +545,29 @@ TEST_CASE("[Region] Parsing opcodes")
 
     SECTION("pan_oncc")
     {
-        REQUIRE(region.modifiers[Mod::pan].empty());
+        const ModKey target = ModKey::createNXYZ(ModId::Pan, region.getId());
```

Comment:
Why isn't this a CC source? (`createCC`)

Reply:
Not sure of what you mean here, I'll just summarize what goes on.

- Region is going to parseOpcode of everything looking like `pan_onccN` as well as curve/step/etc
  These virtual connections for the region are cached into a list.
  The connection is made later at Synth level, because CC sources are not definitive in an incremental process.
  (example you can receive a smoothcc/curve/step/value for this connection, this will modify its source key)

- As result, you get a bunch of connections in a list
  ```
  cc45 -> pan
  cc17 -> pan
  [...]
  ```
  It's not possible ending with several connections of the same CC number and destination
  ```
  cc45 smooth=10 -> pan
  cc45 smooth=50 -> pan  /// not possible
  ```

- So when you specify a region, target id, cc number, you designate an unique item in there.
  `RegionCCView` gives an interface to access this, while not much modifying too much the structure of the old unit test. (think of it like CCMap in O(n))